cmd/insertvariantobject/Makefile

```diff
@@ -0,0 +1,16 @@
+include ../../gosnowflake.mak
+CMD_TARGET=insertvariantobject
```

Comment:
Can we make it more generic - just naming it `variant`?

Reply:
i'm following naming convention of all the rest of the stuff under `cmd` but of course can change it